src/components/block/DonationProjectCard/DonationProjectCard.module.css

```diff
@@ -0,0 +1,33 @@
+.container {
+  height: 240px;
+  position: relative;
+}
+
+.bottombox {
```

Comment:
One thing I noticed after you added in a real picture (thanks, btw!) is that when an image is a bit too bright, it becomes more difficult to read white text on top of it. That caused me to take a look at the [design in Figma](https://www.figma.com/file/HLVHC58KwkcVFXkKAuY0ej/ShelterTech.org-Design-System?node-id=665%3A2) a bit more closely to see that there's actually a small, partially transparent background gradient on this bottom element, which helps darken the background just below the text and make it more readable. You should also add that background gradient.

Reply:
👍 Gradient added. Implement flex-grow to fill the remainder of the flexbox child with the gradient.